mscore/exportxml.cpp

```diff
@@ -5024,7 +5024,7 @@ void ExportMusicXml::write(QIODevice* dev)
       xml.setDevice(dev);
       xml.setCodec("UTF-8");
       xml << "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n";
-      xml << "<!DOCTYPE score-partwise PUBLIC \"-//Recordare//DTD MusicXML 3.0 Partwise//EN\" \"http://www.musicxml.org/dtds/partwise.dtd\">\n";
+      xml << "<!DOCTYPE score-partwise PUBLIC \"-//Recordare//DTD MusicXML 3.1 Partwise//EN\" \"http://www.musicxml.org/dtds/partwise.dtd\">\n";
```

Comment:
I would keep it if 3.1 is entirely compatible with 3.0 and I believe it's the case.